src/Cids.sol

```diff
@@ -1,8 +1,16 @@
-// SPDX-License-Identifier: Apache-2.0 OR MIT
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
BRO

Reply:
accidental left-over from me checking out this file from another branch, going to fix